deploy/rbac-clowdapp.yml

```diff
@@ -181,6 +181,8 @@ objects:
             value: ${NOTIFICATIONS_TOPIC}
           - name: EXTERNAL_SYNC_TOPIC
             value: ${EXTERNAL_SYNC_TOPIC}
+          - name: SHOULD_MIGRATE
```

Comment:
As this looks that it will be used only for init container - maybe it would be nice that name would contain that fact, something like:
`INIT_MIGRATE`

Other fact is there is also seed process and not only migrate but with this I am not sure how to name it :)
something like `MIGRATE_AND_SEED_ON_INIT` 

maybe you will have better idea :) Let me know what do you think, thanks!